firmware/arduino/chessboard/motorDriver.ino

```diff
@@ -49,42 +49,55 @@ void disableMotors()
 // Drives the motor corresponding to "motor" to it's home position (position 0)
 void homeAxis(int motor[])
 {
+  int * currentMotorPos;
+  int currentMotorDir, currentMotorNumEighthSteps;
   int i;
 
+  currentMotorPos = (motor == xMotor) ? &currentX : &currentY;
```

Comment:
Please add a comment explaining that this is storing the corresponding motor position so we can increment the right one